ui/raidboss/data/06-ew/dungeon/the_tower_of_zot.txt

```diff
@@ -20,11 +20,11 @@ hideall "--sync--"
 27.8 "Manusya Thunder III x7" sync / 1[56]:[^:]*:Minduruva:6293:/
 32.9 "Manusya Bio III" sync / 1[56]:[^:]*:Minduruva:6298:/
 34.1 "--sync--" sync / 1[56]:[^:]*:Minduruva:6299:/
-38.0 "Transmute Thunder III" sync / 1[56]:[^:]*:Minduruva:631C:/
+38.0 "Transmute Element III" sync / 1[56]:[^:]*:Minduruva:6(29A|31[BCD]):/
 38.9 "--sync--" sync / 1[56]:[^:]*:Minduruva:629B:/
 42.3 "Manusya Fire" sync / 1[56]:[^:]*:Minduruva:6463:/
-47.7 "Manusya Blizzard III" sync / 1[56]:[^:]*:Minduruva:6292:/
-51.8 "Delayed Thunder III" #sync / 1[56]:[^:]*:Minduruva:6297:/
+47.7 "Manusya Element III" #sync / 1[56]:[^:]*:Minduruva:629[5678]:/
+51.8 "Delayed Element III" #sync / 1[56]:[^:]*:Minduruva:629[5678]:/
```

Comment:
Looking at this now, I kind of wish I had written these syncs out as `(6295|6296|6297|6298)`, just to make searching for the individual ability IDs easier.

Reply:
Sure, done!